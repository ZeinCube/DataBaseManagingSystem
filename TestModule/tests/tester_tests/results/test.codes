{"status":"OK","command":"hello;","answer":"Connected to port 3309"}
{"status":"PARSER_ERROR","command":"world;","answer":"line 1:0 mismatched input 'hello' expecting {K_CREATE, K_DROP, K_SHOW, K_SELECT, K_UPDATE, K_INSERT}"}
{"status":"PARSER_ERROR","command":"foo;","answer":"line 1:0 mismatched input 'world' expecting {K_CREATE, K_DROP, K_SHOW, K_SELECT, K_UPDATE, K_INSERT}"}
{"status":"PARSER_ERROR","command":"bar;","answer":"line 1:0 mismatched input 'foo' expecting {K_CREATE, K_DROP, K_SHOW, K_SELECT, K_UPDATE, K_INSERT}"}
