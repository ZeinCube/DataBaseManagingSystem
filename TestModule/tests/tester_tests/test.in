[@PrintLevel] EXTENDED
hello;
@@ Example comment
[@Repeat] 5
world$i;
[@Sleep] 500
[@Clear]
foo;
bar;