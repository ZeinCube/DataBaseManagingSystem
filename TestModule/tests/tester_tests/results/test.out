Connected to port 3309
line 1:0 mismatched input 'hello' expecting {K_CREATE, K_DROP, K_SHOW, K_SELECT, K_UPDATE, K_INSERT}
line 1:0 mismatched input 'world' expecting {K_CREATE, K_DROP, K_SHOW, K_SELECT, K_UPDATE, K_INSERT}
line 1:0 mismatched input 'foo' expecting {K_CREATE, K_DROP, K_SHOW, K_SELECT, K_UPDATE, K_INSERT}
